.github/functions.sha256

```diff
@@ -0,0 +1 @@
+02f134cc72be74a8a37d0750a42f667fa5da3d52ca9aa16912162f3b0ecf2724  .github/functions
```

Comment:
No, this value `02f134cc72be74a8a37d0750a42f667fa5da3d52ca9aa16912162f3b0ecf2724` needs to be hardcoded in the workflow for it to be secure. Reading it from a file in the workspace at runtime negates the security.